components/net/fetch/cors_cache.rs

```diff
@@ -48,7 +49,7 @@ pub struct CorsCacheEntry {
     pub max_age: u32,
```

Comment:
I think making `max_age` a `Duration` here will greatly simplify the code and make the units clearer.